test/OpenTelemetry.Tests/Internal/GuardTest.cs

```diff
@@ -171,5 +163,38 @@ public void ZeroTest()
             Assert.Contains("Must not be zero", ex1.Message);
             Assert.Equal("0", ex1.ParamName);
         }
+
+        public class Thing
+        {
+            public string Bar { get; set; }
+        }
+
+        /// <summary>
+        /// Borrowed from: <see href="https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime/tests/System/Runtime/CompilerServices/CallerArgumentExpressionAttributeTests.cs"/>.
+        /// </summary>
+        public class CallerArgumentExpressionAttributeTests
+        {
+#if !NETCOREAPP3_0_OR_GREATER
+        [Theory]
```

Comment:
The dotnet-format CI is angry with me here.
It's not apparent what needs to be changed.

> D:\a\opentelemetry-dotnet\opentelemetry-dotnet\test\OpenTelemetry.Tests\Internal\GuardTest.cs(176,10): error WHITESPACE: Fix whitespace formatting. Replace 2 characters with '\r\n'. [D:\a\opentelemetry-dotnet\opentelemetry-dotnet]
> D:\a\opentelemetry-dotnet\opentelemetry-dotnet\test\OpenTelemetry.Tests\Internal\GuardTest.cs(17[8](https://github.com/open-telemetry/opentelemetry-dotnet/runs/6988755470?check_suite_focus=true#step:5:9),9): error WHITESPACE: Fix whitespace formatting. Insert '\s\s\s\s'. [D:\a\opentelemetry-dotnet\opentelemetry-dotnet]

Reply:
Agree the errors are confusing. Though just looking at this definitely seems some indentation is off.

From `[Theory]` on down needs another four space indent.

Probably not a problem, but might also move `#if !NETCOREAPP3_0_OR_GREATER` to outside the class definition.

Reply:
Thanks, the indention was exactly the problem.
Because of the preprocessor, the format document wasn't working. 🤦‍♂️

I initially did have the preprocessor outside of the class, but then the formatter was identifying the xml comment `///` as an invalid formatted comment `//`. 

I got busy with other work this week. I should be able to circle back to this by Monday.

Reply:
Fixed the preprocessor.